sdk/communication/Azure.Communication.Calling.Server/src/Generated/ConversationRestClient.cs

```diff
@@ -129,6 +129,90 @@ public Response<JoinCallResponse> JoinCall(string conversationId, JoinCallReques
             }
         }
 
+        internal HttpMessage CreatePlayAudioRequest(string conversationId, PlayAudioRequest request)
+        {
+            var message = _pipeline.CreateMessage();
+            var request0 = message.Request;
+            request0.Method = RequestMethod.Post;
+            var uri = new RawRequestUriBuilder();
+            uri.AppendRaw(endpoint, false);
+            uri.AppendPath("/calling/conversations/", false);
+            uri.AppendPath(conversationId, true);
+            uri.AppendPath("/PlayAudio", false);
+            uri.AppendQuery("api-version", apiVersion, true);
+            request0.Uri = uri;
+            request0.Headers.Add("Accept", "application/json");
+            request0.Headers.Add("Content-Type", "application/json");
+            var content = new Utf8JsonRequestContent();
+            content.JsonWriter.WriteObjectValue(request);
+            request0.Content = content;
+            return message;
+        }
+
+        /// <summary> Play audio in a call. </summary>
+        /// <param name="conversationId"> The conversation id which can be guid or encoded cs url. </param>
+        /// <param name="request"> Play audio request. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="conversationId"/> or <paramref name="request"/> is null. </exception>
+        public async Task<Response<PlayAudioResponse>> PlayAudioAsync(string conversationId, PlayAudioRequest request, CancellationToken cancellationToken = default)
+        {
+            if (conversationId == null)
+            {
+                throw new ArgumentNullException(nameof(conversationId));
+            }
+            if (request == null)
+            {
+                throw new ArgumentNullException(nameof(request));
+            }
+
+            using var message = CreatePlayAudioRequest(conversationId, request);
+            await _pipeline.SendAsync(message, cancellationToken).ConfigureAwait(false);
+            switch (message.Response.Status)
+            {
+                case 202:
```

Comment:
is message.Response.Status the HttpStatus enum? If so, use the enum in the case statement.

Reply:
Okay, I checked its not `enum` but `int`. Also I believe these are auto generated from the swagger not sure if should update any `Generated/` files